ui/components/issuessection/issuessection.go

```diff
@@ -89,6 +89,12 @@ func (m Model) Update(msg tea.Msg) (section.Section, tea.Cmd) {
 				if msg.NewComment != nil {
 					currIssue.Comments.Nodes = append(currIssue.Comments.Nodes, *msg.NewComment)
 				}
+				if msg.AddedAssignees != nil {
+					currIssue.Assignees.Nodes = append(currIssue.Assignees.Nodes, msg.AddedAssignees.Nodes...)
```

Comment:
Could this cause duplicates to appear if I try to add a user that is already assigned to the issue?

Reply:
oh, great catch!

yes, it does render duplicates - at least until you refresh and it loads the correct list from GitHub.

I'll fix this. Thanks for catching it.